.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Documentation & Website
+about: Issues regarding documentation and website
+title: '[DOC/WEBSITE] '
+labels: Doc, Website, needs triage
+assignees: ''
+
+---
+
+## Current state
+Link: (Insert link to location where faulty documentation is located)
+
+(Describe what you are currently seeing in the faulty state or insert a screenshot)
+
+## Expected state
+(Describe what you'd rather expect)
```

Comment:
the information here is insufficient to resolve issues, as essential information such as browser, systems, active plugins are missing. maybe start with bug template and adjust as needed?